src/common/replayBrowser/dao.ts

```diff
@@ -0,0 +1,183 @@
+import { FileResult } from "common/replayBrowser";
+import { app } from "electron";
+import path from "path";
+import * as sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(path.join(app.getPath("userData"), "statsdb.sqlite"), (err) => {
```

Comment:
i think this is a better name
```suggestion
const db = new sqlite3.Database(path.join(app.getPath("userData"), "sqlippi.db"), (err) => {
```